pyproject.toml

```diff
@@ -0,0 +1,121 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "invoke"
+version = "2.0.0"
+description = "Pythonic task execution"
+# keywords = ["automation", "task runner"]
+authors = [{author="Jeff Forcier", email="jeff@bitprophet.org"}]
+maintainers = [{author="Jeff Forcier", email="jeff@bitprophet.org"}]
+requires-python = ">=3.6.2"
+readme = "README.rst"
+license = {file = "LICENSE"}
+classifiers=[
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: System :: Software Distribution",
+    "Topic :: System :: Systems Administration",
+]
+
+dependencies = [
+    "importlib-metadata>=2.1.1"
+]
+
+[project.optional-dependencies]
+dev = [
+    # build
+    "build",
```

Comment:
Why is `build` a `dev` requirement? Shouldn't you be using `pip` commands moving forward with this.?

Reply:
I suppose this can be removed.